e2e/support/test_tables.js

```diff
@@ -202,3 +202,36 @@ export const multi_schema = async dbClient => {
 
   return schemas;
 };
+
+export const many_schemas = async dbClient => {
```

Comment:
Our `multi_schemas` database only has 2 schemas, and this issue can only be reproduced with way more of them.